.github/actions/test-khiops-install/action.yml

```diff
@@ -1,74 +1,54 @@
 ---
 name: Test Khiops Installation
-description: Tests a Khiops installation on the Iris dataset
+description: Check the outputs of khiops in the console
 runs:
   using: composite
+  # Test if the output of Khiops in the console is as expected (no mpi or java errors)
+  # We expect only one line in stdout+stderr when running 'khiops -v'
   steps:
-    # Python install: we don't use the setup-python action because of the following error:
-    # python3: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by python3)
-    - name: Python setup for Linux
-      if: runner.os == 'Linux'
+    - name: Print status for debugging
       shell: bash
```

Comment:
I don't think we need `bash` here; the default `sh` (`dash` on Ubuntu) shell should suffice IMHO (see the other comment on the same topic).

Reply:
The shell definition is mandatory in actions

Reply:
Yes, right.